examples/tti/tti_example.py

```diff
@@ -55,7 +55,7 @@ def run(dimensions=(50, 50, 50), spacing=(20.0, 20.0), tn=250.0,
     rec, u, v, gflops, oi = TTI.Forward(
         cse=cse, auto_tuning=auto_tuning, cache_blocking=cache_blocking, compiler=compiler
     )
-    return gflops, oi, rec, u, v
+    return gflops, oi, [rec, u, v]
 
```

Comment:
As pointed before, outputing gflops and OI at all time is not necessarly the best way. Makes practical usage more complicated than necessary. Jut my point of view